Tasks/XamariniOS/task.json

```diff
@@ -239,6 +239,8 @@
         "MSB_DetectedNuGetExtensionsLoaderPath0": "Detected NuGet extensions loader path. Environment variable NUGET_EXTENSIONS_PATH is set to '{0}'.",
         "MSB_SolutionNotFoundUsingSearchPattern0": "Solution not found using search pattern '{0}'.",
         "MSB_RestoreNuGetPackagesDeprecated": "The 'Restore NuGet Packages' option is deprecated. To restore NuGet packages in your build, add a NuGet Installer task to your build definition.",
-        "MSB_BuildToolNotFound": "MSBuild or xbuild (Mono) were not found on the macOS or Linux agent."
+        "MSB_BuildToolNotFound": "MSBuild or xbuild (Mono) were not found on the macOS or Linux agent.",
+        "MultipleSolutionsFound": "Multiple solution file matches were found. The first match will be used: %s",
+        "SolutionDoesNotExist": "The solution file was specified, but it does not exist or is not a directory: %s"
```

Comment:
Should be "is a directory"

Reply:
Oops.  Actually I believe if there is a directory named *.sln, the code will still find it and try to build it but will fail.  So i will just remove this part.